src/extension.ts

```diff
@@ -171,6 +172,7 @@ function initCommandAsync(context: vscode.ExtensionContext,
                 properties.errorMessage = errorData.message;
             }
             TelemetryClient.sendEvent(`${commandId}.end`, properties);
+            NSAT.takeSurvey(context);
```

Comment:
for each command registed wil call take survey?

Reply:
Yes

Reply:
trigger command will call take survey